src/operator/src/statement/tql.rs

```diff
@@ -37,12 +37,16 @@ impl StatementExecutor {
                     end: eval.end,
                     step: eval.step,
                     query: eval.query,
+                    lookback: eval.lookback.unwrap_or(DEFAULT_LOOKBACK_STRING.to_string()),
```

Comment:
I'd like to use `eval.lookback.unwrap_or_else(|| DEFAULT_LOOKBACK_STRING.to_string())`.

Avoid an allocation if the lookback value is provided. @etolbakov

Reply:
Good point @killme2008! 
I'll raise a follow-up PR to address that in the next couple of days if that's ok